internal/datasource/webclient/cloudproviders/genqlient.yaml

```diff
@@ -3,7 +3,7 @@
 schema: ../../../api/graphql/schema.graphql
```

Comment:
missed to add one more path down the folders here, it should `schema: ../../../../api/graphql/schema.graphql`

Reply:
Fixed